webui/react/src/pages/TrialDetails/TrialInfoBox.tsx

```diff
@@ -109,24 +87,15 @@ const TrialInfoBox: React.FC<Props> = ({ trial, experiment }: Props) => {
           <OverviewStats title="Checkpoints">{`${trial?.checkpointCount} (${totalCheckpointsSize})`}</OverviewStats>
         )}
         {bestCheckpoint && (
-          <>
-            <OverviewStats title="Best Checkpoint" onClick={handleModalCheckpointClick}>
+          <CheckpointModalTrigger
```

Comment:
nit: if this is the only place we're using `CheckpointModalTrigger` now, maybe we don't really need it and could just go with the pattern with `useCheckpointFlow` everywhere?

Reply:
That's fair!